README.md

```diff
@@ -10,7 +10,9 @@
 
 This repository is the frontend UI for accessing and exploring the openly-licensed content provided by the [Openverse API](https://github.com/wordpress/openverse-api).
 
-The application is live at its previous home: [search.creativecommons.org](https://search.creativecommons.org/).
+The application is live at its previous home: [wordpress.org/openverse](https://wordpress.org/openverse).
+
+We publish our Storybook live at [wordpress.github.io/openverse-frontend](https://wordpress.github.io/openverse-frontend)
```

Comment:
Recommend making the text itself the link. A very crude example would be something like so.
```suggestion
You can view [the application](https://wordpress.org/openverse) live on WordPress.org. You can also view [the production  Storybook](https://wordpress.github.io/openverse-frontend)
```

If we're updating the link to CC Search to point to WordPress.org, we should not call it "previous home".